scripts/dex-router/src/router/jupiter.ts

```diff
@@ -5,17 +5,18 @@ import {
   Instruction as JupiterInstruction,
 } from "@jup-ag/api";
 
-const maxAccounts = 20;
-const maxAutoSlippageBps = 50;
+const MAX_SLIPPAGE_BPS = 50;
 
 export class JupiterRouter implements Router {
   private chainId: string;
   private executor: PublicKey;
+  private maxAccounts: number;
   private jupiterClient = createJupiterApiClient();
 
-  constructor(chainId: string, executor: PublicKey) {
+  constructor(chainId: string, executor: PublicKey, maxAccounts: number = 20) {
```

Comment:
I think this is an important variable and should not have a default value so that we force the user to decide about it.

Reply:
this is fair. 20 is a reasonable value i often found to work, so that's why i wanted to leave it as a default as a hint to the user. otherwise, without context there's no way for a user to know what's reasonable without a bunch of trial and error

Reply:
good point